index.js

```diff
@@ -1,6 +1,4 @@
 /* eslint-disable class-methods-use-this */
-/* eslint-disable max-classes-per-file */
```

Comment:
- [ ] Let's try to not disable Linter checks on your codebase. Kindly fix the error as the Linter checks are there to help you write correct and professional code. Please fix this 👍🏽